src/hooks/useSwapCallback.ts

```diff
@@ -164,9 +164,14 @@ export function useSwapCallback(
                   }`
 
             const minerBribeBN = BigNumber.from(args[1])
-            const estimatedEffectiveGasPriceBn = minerBribeBN.div(BigNumber.from(trade.estimatedGas))
+            const totalFees = minerBribeBN
```

Comment:
don't really need this change anymore if the rest stays commented out.

Reply:
It will be uncommented if flashbots merges https://github.com/flashbots/mev-geth/tree/mev-callbundle